students/juyeob/westagram/users/views.py

```diff
@@ -1,2 +1,44 @@
-# Create your views here.
+
 import json
+import re
+
+from django.http  import JsonResponse
+from django.views import View
+
+from users.models import User
+
+class SignUpView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+
+        email           = data['email']
+        password        = data['password']
+
+        try:
+            if not re.match(
+                '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$', email):
+                return JsonResponse({"message": "ERROR_EMAIL_VALIDATION"}, status=400)
```

Comment:
1. = 기준으로 정렬할 때 가장 긴 단어를 기준으로 1칸만 띄워 주세요.
2. data 변수부터 try에 넣어 주세요.
3.  그리고 왜 email과 password만 위에서 먼저 선언하셨나요? 나머지 값들도 위에서 선언해 주세요. (그리고 왜 위에서 선언하면 좋을지도 생각해 보시고 코멘트로 남겨주세요.)
```suggestion
        try:
            data = json.loads(request.body)

            email    = data['email']
            password = data['password']
        
            if not re.match(
                '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$', email):
                return JsonResponse({"message": "ERROR_EMAIL_VALIDATION"}, status=400)
```